posw/src/lib.rs

```diff
@@ -117,14 +117,14 @@ mod tests {
 
     #[test]
     fn test_posw_marlin() {
-        let rng = &mut XorShiftRng::seed_from_u64(1234567);
+        let rng = &mut ChaChaRng::seed_from_u64(1234567);
```

Comment:
Warning: This change causes the CI time for posw to go up a lot.

Reply:
NOPE <- I guess ChaChaRng is not the reason. There is likely another reason why posw test becomes more expensive.

Reply:
It's likely due to the overpaying of to_bytes_le.

Reply:
I guess we can handle this in a separate PR. Let me think about a way to avoid `to_bytes_le`. Ideally we want some sort of zero-copying here. 

The easiest solution is this: We only have Marlin SRS so far, so we can just...use MarlinSRS. But I need to handle the trait parameters.

Reply:
Solved in #301. 10m now down to 3m.